interhost.py

```diff
@@ -204,6 +204,60 @@ def parser_snpEff(parser=argparse.ArgumentParser()):
 __commands__.append(('snpEff', parser_snpEff))
 
 
+# =======================
+# ***  align_mafft  ***
+# =======================
+
+def parser_align_mafft(parser=argparse.ArgumentParser()):
+    parser.add_argument('inFastas', nargs='+',
+        help='Input FASTA files.')
+    parser.add_argument('outFile',
+        help='Output file containing alignment result (default format: FASTA)')
+    parser.add_argument('--localpair', default=None, action='store_true',
```

Comment:
--localpair and --globalpair can be a mutually exclusive group:
https://docs.python.org/3/library/argparse.html#mutual-exclusion
(this doesn't negate the need to check within the Tool.execute though, as you are already doing)